### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by t-lohse**
Settings sent through both gRPC and the CLI, along with some logging and clippy warning removal
In `src/logging.rs`, I've implemented a logging protocol for logging specific information, but as of right now it only works on either the CLI, or with a single client. We have assesed that this is out of scope for this project/does not fit any areas of work, so I've placed a TODO, and the Ecdar organization should themselves implement a logging that works for several clients

### CODE REVIEW

**Review by HTHNLNDN — APPROVED** [review]

File: src/logging.rs [hunk 1]
```diff
@@ -1,31 +1,107 @@
...
+                ColorChoice::Auto,
+            )
+        },
+    ];
+
+    CombinedLogger::init(loggers)
+}
+
+fn get_messages_raw() -> impl Iterator<Item = String> + Sized {
+    unsafe {
```
Comment (henneboy) [inline src/logging.rs:43]: Can this be something other than unsafe?

Comment (t-lohse) [reply src/logging.rs:43]: no, when dealing with `static mut` it must be in an `unsafe` function or block

Comment (henneboy) [reply src/logging.rs:43]: Is it possible to use something other than `static mut`?

Comment (t-lohse) [reply src/logging.rs:43]: Not really

**Review by henneboy — APPROVED** [review]
Lgtm, maybe make an issue about making this code not use unsafe code or add a mutex to make it safe for multithreading

**Comment by t-lohse** [comment]
@henneboy @HTHNLNDN I have removed the logger, I changed my mind and think that it should just be `unimplemented`

**Review by henneboy — APPROVED** [review]
Lgtm

**Review by lugialukas — APPROVED** [review]
LGTM

**Comment by mads256h** [comment]
plx dont approve uncommented code :(

**Pull request merged**